packages/react-components/src/components/Loader/Loader.stories.tsx

```diff
@@ -23,3 +23,13 @@ export const labeled = (args: LoaderProps): React.ReactElement => (
 export const customColors = (args: LoaderProps): React.ReactElement => (
   <Loader primaryColor="#d64646" secondaryColor="#eec4c5" {...args} />
 );
+
+export const coverSize = (args: LoaderProps): React.ReactElement => {
```

Comment:
```suggestion
export const coverSize = (args: LoaderProps): ReactElement => {
```

Maybe we can adjust imports from `react` because we don't need to import all in v17

Reply:
Cover size is removed